tee-worker/app-libs/stf/src/trusted_call.rs

```diff
@@ -672,6 +667,14 @@ where
 	pallet_sudo::Pallet::<Runtime>::key().map_or(false, |k| account == &k)
 }
 
+pub(crate) fn convert_stf_error(e: StfError) -> IMPError {
+	match e {
+		StfError::Dispatch(s) =>
+			IMPError::StfError(ErrorString::truncate_from(s.as_bytes().to_vec())),
+		_ => IMPError::StfError(ErrorString::truncate_from(format!("{:?}", e).as_bytes().to_vec())),
+	}
+}
+
```

Comment:
This converter is not needed I think.

`StfError` implements `Display` and thus `to_string`, we can simply use `e.to_string()` => for `StfError::Dispatch` too.

I suggest we just implement:
```
impl StfError {
  pub fn to_imp_error(&self) ....
}
```

What do you think?

Reply:
a following issue for ts-test is created: https://github.com/litentry/litentry-parachain/issues/1374

Reply:
fixed